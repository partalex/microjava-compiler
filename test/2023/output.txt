966bc-4
Completion took 1 ms
