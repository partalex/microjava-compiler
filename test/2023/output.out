01v97
4
15
12243036
2772126
3036
126
243036

Completion took 9 ms
